src/core/dbus-manager.c

```diff
@@ -1451,6 +1451,30 @@ int verify_run_space_and_log(const char *message) {
         return 0;
 }
 
+static void log_reload_caller(sd_bus_message *message, Manager *manager) {
+        _cleanup_(sd_bus_creds_unrefp) sd_bus_creds *creds = NULL;
+        Unit *caller;
+        pid_t pid;
+        int r;
+
+        assert(message);
+        assert(manager);
+
+        r = sd_bus_query_sender_creds(message, SD_BUS_CREDS_PID, &creds);
+        if (r < 0)
+                return;
```

Comment:
Not a matter, but if you do not use the return value, then you can shorten the code a bit.
```c
if (sd_bus_query_sender_creds(...) < 0)
        return;
```